tests/std/tests/P0896R4_views_drop/test.cpp

```diff
@@ -16,6 +16,12 @@ using namespace std;
 
 #pragma warning(disable : 6011) // Dereferencing NULL pointer '%s'
 
+struct convertible {

```

Comment:
Can we:
1. make this class explicitly `move_constructible`,
2. implement the other SMFs, but have them explode, and
3. give it a less generic name like `evil_length_type`?